CHANGELOG.md

```diff
@@ -48,6 +48,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- [#4309](https://github.com/firecracker-microvm/firecracker/pull/4309): The
```

Comment:
It is changed or fixed the right section? before we were ignoring the option right?

Reply:
I put it in **changed** because I don't think the previous behavior was a bug, as that was not documented to work. Although I marked the commit as `fix`, so I was inconsistent :/.

Reply:
yes, I agree this is a bit misleading. If we don't think it is a bug, we should probably rename the commit.

Reply:
Renamed the commit to `feat(jailer)`